util/util.go

```diff
@@ -883,3 +883,11 @@ func IsKubernetesVersionAtLeast(kubeClient clientset.Interface, vers string) (bo
 	minVersion := version.MustParseSemantic(vers)
 	return currentVersion.AtLeast(minVersion), nil
 }
+
+func StringToStringList(s string) ([]string, error) {
```

Comment:
Wrong name. And why do we need this function? It's just one line of unmarshalling a JSON string.